addons/mail/static/src/core/common/message.xml

```diff
@@ -177,4 +178,10 @@
     </div>
 </t>
 
+<t t-name="mail.Message.notification.body">
+    <div class="o-mail-Message-body text-break mb-0 w-100">
+        <t t-out="messageSearch?.highlight(message.body) ?? message.body"/>
+    </div>
+</t>
```

Comment:
`mail.Message.notification.body` => `mail.Message.bodyAsNotification`

Reasoning: keep name small and straight-forward, as `mail.ComponentName.partName`.